src/PhpImap/Mailbox.php

```diff
@@ -850,7 +850,7 @@ protected function initMailPart(IncomingMail $mail, $partStructure, $partNum, $m
 		if(!empty($partStructure->parameters)) {
 			foreach($partStructure->parameters as $param) {
 				if (!isset($param->value) || empty($param->value)) {
-					continue;
+					$params[strtolower($param->attribute)] = $param->value;
 				}
 				
 				$params[strtolower($param->attribute)] = $this->decodeMimeStr($param->value);
```

Comment:
This needs to be in a `else`-clause now as you do not skip (using `continue;`) it anymore.

Right now, not set values or empty values would be just set and after it will be decoded although we want to avoid it.